models/demos/resnet/tests/test_metal_resnet50.py

```diff
@@ -118,7 +118,7 @@
 
 
 @skip_for_wormhole_b0("This test is not supported on WHB0, please use the TTNN version.")
-@pytest.mark.parametrize("batch_size", [1, 2, 8, 16, 20], ids=["batch_1", "batch_2", "batch_8", "batch_16", "batch_20"])
```

Comment:
Can you also add a NOTE saying that batch 1 and 2 are not yet supported with sharded data?